src/nvim/decoration.c

```diff
@@ -490,18 +490,21 @@ static void decor_range_add_from_inline(DecorState *state, int start_row, int st
   if (decor.ext) {
     DecorVirtText *vt = decor.data.ext.vt;
     while (vt) {
-      decor_range_add_virt(state, start_row, start_col, end_row, end_col, vt, owned);
+      decor_range_add_virt(state, start_row, start_col, end_row, end_col, vt, owned,
+                           DECOR_PRIORITY_BASE);
```

Comment:
If I understand the decor code correctly (very likely that I do not) then this code path is only taken for non-ephemeral marks, in which case subpriorites are unused/ignored, so we just pass the default value.